index.html

```diff
@@ -24,71 +24,449 @@
+              alt="counter shape"
+            />
+            <h6 class="title-year">2022</h6>
+          </section>
+          <p class="grid-item-two-body-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+
+          <ul class="js-css-html-button-ul">
+            <li class="js-css-html-button-li">
+              <button type="button" class="html-css-js-button">html</button>
+            </li>
+            <li class="js-css-html-button-li">
+              <button type="button" class="html-css-js-button">css</button>
+            </li>
+            <li class="js-css-html-button-li">
+              <button type="button" class="html-css-js-button">
+                javascript
+              </button>
+            </li>
+          </ul>
+
+          <button type="button" class="footer-button">
+            <i class="see-project-text">See Project</i>
+          </button>
+        </section>
+      </div>
+    </div>
+    <div class="grid-item">
+      <div class="grid-item-three">
+        <header class="grid-item-three-header">
+          <img
+            src="images/SnapshootPortfolio5.svg"
+            alt="A snapshoot Portfolio Header From Figma"
+          />
+        </header>
+        <section>
+          <h2 class="grid-item-two-body-title">Multi-Post Stories</h2>
+
+          <section class="subhead-title-year">
+            <h4 class="grid-item-two-body-subhead">CANOPY</h4>
+            <img
+              class="counter-image"
+              src="images/Counter.svg"
+              alt="counter shape"
+            />
+            <h6 class="title-year">Front End Dev</h6>
+            <img
+              class="counter-image"
+              src="images/Counter.svg"
+              alt="counter shape"
+            />
+            <h6 class="title-year">2022</h6>
+          </section>
+          <p class="grid-item-two-body-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+
+          <ul class="js-css-html-button-ul">
+            <li class="js-css-html-button-li">
+              <button type="button" class="html-css-js-button">html</button>
+            </li>
+            <li class="js-css-html-button-li">
+              <button type="button" class="html-css-js-button">css</button>
+            </li>
+            <li class="js-css-html-button-li">
+              <button type="button" class="html-css-js-button">
+                javascript
+              </button>
+            </li>
+          </ul>
+
+          <button type="button" class="footer-button">
+            <i class="see-project-text">See Project</i>
+          </button>
+        </section>
+      </div>
+    </div>
+    <div class="grid-item">
+      <div class="grid-item-last-two-pages">
+        <div class="about-myself-div">
+          <h1 class="text-about-myself">About Myself</h1>
```

Comment:
- [ ] It would be better to use a header tag from h2 -h6, your page should have only one h1 tag, this makes it easy for screen reader to read your website  👍🏿